bin/phpactor.php

```diff
@@ -1,38 +0,0 @@
-<?php
-
-use Symfony\Component\Console\Input\ArgvInput;
-
-$autoloadFile = __DIR__ . '/../vendor/autoload.php';
-if (!file_exists($autoloadFile)) {
-    echo sprintf(
-        'Phpactor dependencies not installed. Run `composer install` (https://getcomposer.org) in "%s"' . PHP_EOL,
-        realpath(__DIR__ . '/..')
-    );
-    exit(255);
-}
-
-require_once($autoloadFile);
-
-$minVersion = '7.0.0';
-
-if (version_compare(PHP_VERSION, $minVersion) < 0) {
-    echo 'Phpactor requires at least PHP ' . $minVersion;
-    exit(255);
-}
```

Comment:
I think this version check is missing from the new version?

Reply:
> I also removed the version check for php 7.0, as the version constraint in composer should be enough.

But then I can put it back, but IMO it is not really worth it

Reply:
Yes please. Composer won't stop Phpactor being executed with PHP < 7 (e.g. when you change your default version of PHP, or when executed from a PHAR).

Reply:
Done